Brianoflondon adding v4vapp

brianoflondon opened the pull request:
Adding my v4v.app to the Finance Apps

stoodkev commented:
Until we have a better way to show dApps to the users, they are added following the FIFO rule, please put your dApp to the bottom of the list. 🙏

brianoflondon commented:
> Until we have a better way to show dApps to the users, they are added following the FIFO rule, please put your dApp to the bottom of the list. 🙏

I've moved my item down and updated my branch, I hope that's reflected here on this pull request?
